src/groups/mqb/mqbc/mqbc_clusterutil.cpp

```diff
@@ -785,6 +798,8 @@ void ClusterUtil::populateQueueAssignmentAdvisory(
     BSLS_ASSERT_SAFE(domain);
     BSLS_ASSERT_SAFE(clusterData->electorInfo().isSelfActiveLeader());
 
+    BALL_LOG_SET_CATEGORY(k_LOG_CATEGORY);
```

Comment:
Do we need it (considering `BALL_LOG_SET_CLASS_CATEGORY("MQBC.CLUSTERUTIL");)`?
And below

Reply:
Good catch, these lines are redundant. Will remove them.